src/data/download_links.yml

```diff
@@ -1,73 +1,73 @@
-release_notes: "https://github.com/companyzero/bisonrelay/releases/tag/v0.2.1"
+release_notes: "https://github.com/companyzero/bisonrelay/releases/tag/v0.2.2"
 andr:
   gui:
     msg: "Android APK File"
     links:
       -
         os: Android
         arch: arm64
-        url: "https://github.com/companyzero/bisonrelay/releases/download/v0.2.1/bisonrelay-android-v0.2.1.apk"
+        url: "https://github.com/companyzero/bisonrelay/releases/download/v0.2.2/bisonrelay-android-v0.2.2.apk"
 
 macos:
   cli:
     msg: "CLI client for macOS"
-    example_filename: "brclient-darwin-amd64-v0.2.1"
+    example_filename: "brclient-darwin-amd64-v0.2.2"
     links:
       -
         os: macOS
         arch: amd64
-        url: "https://github.com/companyzero/bisonrelay/releases/download/v0.2.1/brclient-darwin-amd64-v0.2.1"
+        url: "https://github.com/companyzero/bisonrelay/releases/download/v0.2.2/brclient-darwin-amd64-v0.2.2"
       -
         os: macOS
         arch: arm64
-        url: "https://github.com/companyzero/bisonrelay/releases/download/v0.2.1/brclient-darwin-arm64-v0.2.1"
+        url: "https://github.com/companyzero/bisonrelay/releases/download/v0.2.2/brclient-darwin-arm64-v0.2.2"
     
   gui:
     msg: "GUI client for macOS"
     links:
       -
         os: macOS
         arch: amd64
-        url: "https://github.com/companyzero/bisonrelay/releases/download/v0.2.1/BisonRelay-darwin-amd64-v0.2.1.dmg"
+        url: "https://github.com/companyzero/bisonrelay/releases/download/v0.2.2/BisonRelay-darwin-amd64-v0.2.2.dmg"
```

Comment:
404 on this one

Reply:
ah yah we don't have that amd64 anymore, will remove